### PR TITLE
Polyfill JS features

### PR DESCRIPTION
**Pull request by Velenir**
Polyfills some JS features based on `package.json::browserlist`
We can discuss what exactly we need in that `browserlist`

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr579--dexreact.review.gnosisdev.com

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "src/index.js",
   "sideEffects": false,
+  "browserslist": "> 5%, iOS >= 9, not dead",
```
Comment (alfetopito) [inline package.json]: `not dead` ??

**Comment by biocom** [comment]
iOS 9 Safari:
Screenshot 2020-02-20 at 16 06 13

**Comment by anxolin** [comment]
It's working for me in both safari for Mac and IOS. Some pics: 

**Mac: Version 13.0.1 (13608.2.11.1.10)**
image



**IOS: iPhone 11, using Wallet Connect. IOS 13.3.1**
IMG_2966
IMG_2967
IMG_2968
IMG_2967
IMG_2968

**Comment by biocom** [comment]
The problematic iOS's are at least iOS 9, 10 and 11 @anxolin

**Comment by Velenir** [comment]
Don't merge
The problems with 
> The problematic iOS's are at least iOS 9, 10 and 11

File: package.json (hunk 1 above)
Comment (Velenir) [reply package.json]: that's how browserlist specifies it, filters out some ie versions I imagine

**Comment by Velenir** [comment]
@biocom if you could once again check this out in Safari, would be great

**Comment by biocom** [comment]
Will test.

**Comment by biocom** [comment]
👉**URL Tested:** 
https://pr579--dexreact.review.gnosisdev.com

❌**iOS 8 - Safari:**
Screenshot 2020-03-23 at 11 42 13


❌**iOS 9 - Safari:**
Screenshot 2020-03-23 at 11 37 37

❌**Safari 9.1 - DESKTOP Mac El Capitan:**
Screenshot 2020-03-23 at 11 49 40

❌**iOS 11 - Safari:**
Screenshot 2020-03-23 at 11 38 41

✅**iOS 12 - Safari:**
Screenshot 2020-03-23 at 11 40 27

✅**Android 10 - OnePlus 7T - Chrome:**
Screenshot 2020-03-23 at 11 46 42

✅**Android 10 - OnePlus 7T - Firefox (no console log available):**
Screenshot 2020-03-23 at 11 47 34

**Comment by anxolin** [comment]
@Velenir what do we do with this one?

**Comment by Velenir** [comment]
Outdated and unnecessary

**Pull request closed without merging**